applications/connectivity_bridge/src/modules/power_handler.c

```diff
@@ -41,15 +41,15 @@ static bool event_handler(const struct event_header *eh)
 		if (check_state(event, MODULE_ID(main), MODULE_STATE_READY)) {
 			module_active_count = 0;
 
-			k_delayed_work_init(&power_down_work, power_down_handler);
+			k_work_init_delayable(&power_down_work, power_down_handler);
 		} else if (event->state == MODULE_STATE_READY) {
 			module_active_count += 1;
 		} else if (event->state == MODULE_STATE_STANDBY) {
 			module_active_count -= 1;
 			__ASSERT_NO_MSG(module_active_count >= 0);
 
 			if (module_active_count == 0) {
-				k_delayed_work_submit(&power_down_work, K_SECONDS(1));
+				k_work_reschedule(&power_down_work, K_SECONDS(1));
```

Comment:
One or both of the reschedule calls can be converted to `schedule`, depending on what you want the retrigger behavior to be

Reply:
Either should work fine.
I think "reschedule" is what I ideally want: post work item with timeout, but if the work item is already posted due to ready/standby event cycles I want to reset the timeout of the already posted work item. Either way, the timing is not critical for this functionality. If there are two events happening in short succession that's ok.